src/detect-engine-register.h

```diff
@@ -333,6 +333,8 @@ enum DetectKeywordId {
 
     DETECT_AL_JA4_HASH,
 
+    DETECT_AL_FTP_COMMAND,
```

Comment:
lets drop the `AL_` here, I don't like it in other keywords either

Reply:
What do you think about using dynamic IDs for new keywords?

Reply:
Think I'm indifferent about it.

Reply:
Would you like a separate commit with `s/DETECT_AL_/DETECT_/`? everywhere?

Reply:
Sounds good to me.

Reply:
There were so many occurrences I decided to make a single PR with the changes: #12433